store/store_download_test.go

```diff
@@ -1040,3 +1041,36 @@ func (s *storeDownloadSuite) TestTransferSpeedMonitoringWriterUnhappy(c *C) {
 	c.Assert(terr, Equals, true)
 	c.Check(w.Err(), ErrorMatches, "download too slow: .* bytes/sec")
 }
+
+func (s *storeDownloadSuite) TestDownloadTimeoutOnHeaders(c *C) {
+	restore := httputil.MockResponseHeaderTimeout(time.Second)
```

Comment:
Maybe 250ms or 500ms is enough here already?

Reply:
Reduced to 250ms, thanks for the suggestion.